ui/raidboss/data/03-hw/dungeon/the_vault.js

```diff
@@ -90,17 +88,8 @@
     },
     {
       id: 'The Vault Execution',
-      regex: Regexes.headMarker({ id: '0020' }),
-      infoText: function(data, matches) {
-        if (data.me == matches.target) {
-          return {
-            en: 'Spread marker on YOU',
-          };
-        }
-        return {
-          en: 'Avoid ' + data.ShortName(matches.target),
-        };
-      },
+      regex: Regexes.headMarker({ id: '0020', capture: false }),
+      response: Responses.spread(),
```

Comment:
Maybe we need a spreadOn (or antiStackOn?) like we have a stackOn

Reply:
What is the difference between spread and spreadOn? Nobody should overlap in a spread anyway so i dont really see the benefit from spreadOn here?
And if you just want to avoid a single person, shouldn't be something similar to .goAwayFrom("x") be used?

Reply:
Maybe this is awayFrom, then.